config/config.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"io/ioutil"
+	"fmt"
+	"encoding/json"
+	"path"
+	"os"
+	"github.com/8tomat8/SSU-Golang-252-Chat/chat-log"
+)
+
+type Storage struct {
+	Driver string `json:"driver"`
+	Name string `json:"name"`
+}
+
+type Config struct {
+	Storage Storage `json:"storage"`
+}
+
+var config Config
+
+func init() {
+
+	data, err := ioutil.ReadFile(GetFilePath("src/github.com/8tomat8/SSU-Golang-252-Chat/config/config.json"))
+
+	chatlog.HandleError(err, "Can't read config.")
```

Comment:
If you don't handle an error here, you will get panic somewhere else in your application.